tools/buildsteps/ios/configure-xbmc

```diff
@@ -2,4 +2,4 @@ WORKSPACE=${WORKSPACE:-$( cd $(dirname $0)/../../.. ; pwd -P )}
 XBMC_PLATFORM_DIR=ios
 . $WORKSPACE/tools/buildsteps/defaultenv
 
-make -C $WORKSPACE/tools/depends/target/cmakebuildsys
+make -C $WORKSPACE/tools/depends/target/cmakebuildsys CMAKE_EXTRA_ARGUMENTS="-DCMAKE_XCODE_ATTRIBUTE_GCC_WARN_64_TO_32_BIT_CONVERSION=NO"
```

Comment:
no, I've adjusted jenkins to accept the current amount of shorten-64-to-32 warnings

Reply:
ok, ill remove that. felt weird putting it in, but theres no hope of cleaning that warning up anytime soon, and its a standard xcode 10 warning for 64bit targets now.